.github/workflows/sample_build.yml

```diff
@@ -0,0 +1,41 @@
+name: Sample build
+
+on: [push, pull_request]
```

Comment:
It's quite nice to be able to keep an eye on the output, after all, it's part of the user interface of cibuildwheel, and I don't actually see it all that often! Also it's pretty quick to run and I don't think we're short of concurrency on GHA linux containers. So I think I'd like to keep it around. But maybe once I get the other hosting providers working, I'll remove them and just rely on GHA for the sample build though.

(another option could be to run the sample build on the same job as the unit tests, before they start, because most of the time is spent downloading images/pythons, so it wouldn't increase the test runtime much. )

Reply:
OK, yes, that makes sense. So it's not really meant as a true test, but this could be useful :-)